src/main/java/com/conveyal/datatools/manager/utils/ErrorUtils.java

```diff
@@ -0,0 +1,71 @@
+package com.conveyal.datatools.manager.utils;
+
+import com.bugsnag.Bugsnag;
+import com.bugsnag.Report;
+import com.conveyal.datatools.manager.DataManager;
+import com.conveyal.datatools.manager.auth.Auth0UserProfile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class ErrorUtils {
```

Comment:
Should this have a class comment?

Reply:
I'm OK without it since this is a somewhat generic/self-explanatory utils file.

Reply:
Added in 526e9288789f59a0ce19155ba8bec6556738d68e